### PR TITLE
#92: alt attribute on lazy-loaded <picture> tags

### PR DESCRIPTION
**Pull request by dooman87**
This PR is adding data-alt attribute to picture tag, so it can be inserted to the  element of a picture. This is only code changes. If they are fine then I can update README and bump the verision.

### CODE REVIEW

File: dist/lozad.min.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
+/**
+ * Detect IE browser
+ * @const {boolean}
+ * @private
+ */function l(t){t.setAttribute("data-loaded",!0)}var f=function(t){return"true"===t.getAttribute("data-loaded")};return function(){var r,o,a=0<arguments.length&&void 0!==arguments[0]?arguments[0]:".lozad",t=1<arguments.length&&void 0!==arguments[1]?arguments[1]:{},e=s({},g,t),n=e.rootMargin,i=e.threshold,d=e.load,u=e.loaded,c=void 0;return window.IntersectionObserver&&(c=new IntersectionObserver((r=d,o=u,function(t,e){t.forEach(function(t){0<t.intersectionRatio&&(e.unobserve(t.target),f(t.target)||(r(t.target),l(t.target),o(t.target)))})}),{rootMargin:n,threshold:i})),{observe:function(){for(var t,e=(t=a)instanceof Element?[t]:t instanceof NodeList?t:document.querySelectorAll(t),r=0;r<e.length;r++)f(e[r])||(c?c.observe(e[r]):(d(e[r]),l(e[r]),u(e[r])))},triggerLoad:function(t){f(t)||(d(t),l(t),u(t))}}}});
```
Comment (ApoorvSaxena) [inline dist/lozad.min.js]: can you check why this function has been defined separately unlike before?

Comment (ApoorvSaxena) [reply dist/lozad.min.js]: better discard your `.min.js` changes, it'll anyway get overriden by Travis' build

Comment (dooman87) [reply dist/lozad.min.js]: Thanks for reviewing, I rolled back changes in dist/ folder.

**Comment by ApoorvSaxena** [comment]
great work, thanks for your contributions @dooman87 💯

**Pull request merged**